example-conf/mesosphere-dcos/mesos-site.xml

```diff
@@ -44,6 +44,20 @@
     <value>/var/lib/hdfs/data</value>
   </property>
 
+    <!-- Uncomment this to enable secondary data dir
+    <property>
+      <name>mesos.hdfs.secondary.data.dir</name>
+      <description>The secondary data directory in HDFS</description>
+      <value>/var/lib/hdfs/data</value>
+    </property>
+    -->
+    <!--
+    <property>
```

Comment:
We don't need the secondary data directory here even commented out because it's default value is already set in HdfsFrameworkConfig